CodeEditor/Sources/CodeEditor/UXCodeTextView.swift

```diff
@@ -338,6 +352,21 @@ final class UXCodeTextView: UXTextView, HighlightDelegate, UIScrollViewDelegate
         }
         UIGraphicsPopContext()
     }
+    
+    func drawHiglights() {
+        if !text.isEmpty {
+            for hRange in highlightedRanges {
+                self.textStorage.addAttributes(
+                    [
+                        .foregroundColor: UIColor.blue,
+                        .underlineStyle: NSUnderlineStyle.single.rawValue,
+                        .underlineColor: hRange.color,
+                        .link: hRange.id // equals feedback id
+                    ]
+                    , range: hRange.range)
+            }
+        }
+    }
```

Comment:
There will be merge conflicts with #96, which I think we should merge first. This comment is to prevent accidentally merging before we merged #96.

Reply:
#96 is now merged